Client/Extensions/FileManagerExtension.swift

```diff
@@ -143,3 +143,20 @@ extension FileManager.SearchPathDirectory {
         return FileManager.default.urls(for: self, in: .userDomainMask).first
     }
 }
+
+extension FileManager {
+    
+    /// Navigates to download Folder inside the application's folder
+    func openBraveDownloadsFolder() {
+        do {
+            let downloadsPath = try FileManager.default.downloadsPath()
+            
+            let folderPath = downloadsPath.absoluteString.replacingOccurrences(of: "file://", with: "shareddocuments://")
+            guard let braveFolderURL = URL(string: folderPath) else { return }
+
+            UIApplication.shared.open(braveFolderURL)
+        } catch {
+            log.error("Unable to get downloads path: \(error.localizedDescription)")
```

Comment:
Do we want to show an error to the user if for some reason we can't open the folder?

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4769/commits/befde42611abda7f1f9608b9eec5a52bc2a3bca3